sys/riscv/riscv/freebsd64_machdep.c

```diff
@@ -91,7 +91,7 @@ struct sysentvec elf_freebsd_freebsd64_sysvec = {
 	.sv_maxuser	= VM_MAXUSER_ADDRESS,
 	.sv_usrstack	= USRSTACK,
 	.sv_psstrings	= FREEBSD64_PS_STRINGS,
-	.sv_stackprot	= VM_PROT_READ | VM_PROT_WRITE,
+	.sv_stackprot	= VM_PROT_RW_CAP,
```

Comment:
The equivalent mips change has it as VM_PROT_ALL_CAP

Reply:
Also, no similar change for elf_machdep.c?

Reply:
MIPS doesn't do shared page by default so still has an executable stack so it can copy the sigcode out.

In terms of elf_machdep.c, I don't understand why that didn't break CheriABI for me for RISC-V. :(  It should have.  Oh, RISC-V binaries always have `PT_GNU_STACK` so the sysentvec value was never used.  I should still fix elf_machdep.c for RISC-V though.